site/src/pages/api/blocks.api.ts

```diff
@@ -107,11 +66,10 @@ export const readBlocksFromDisk = (): BlockMetadata[] => {
     }))
     .map((metadata: BlockMetadata) => ({
       ...metadata,
+      image: `/blocks/${metadata.packagePath}/${metadata.image}`,
```

Comment:
This assumes that the image URL is relative - can we also handle absolute URLs please?

e.g. check here if it starts with `/`, and if so do this, and if not just include the URL?

Can we also make the same fix for icons?

Reply:
Added in this commit 1146653